### ISSUE TITLE
Accent color resets after system update (Lineage OS)

### ISSUE THREAD

**Issue by FanderWasTaken** [Akilesh-T/ACC#24]
Hello. I've noticed that after installing a system update for Lineage OS (17.1) my accent color goes back to default teal and I have to open the acc app and change the accent manually. Is that something on my end or is it something that can be fixed?

**Comment by Akilesh-T** [comment]
> Hello. I've noticed that after installing a system update for Lineage OS (17.1) my accent color goes back to default teal and I have to open the acc app and change the accent manually. Is that something on my end or is it something that can be fixed?

Does it also happen after a restart?

**Comment by FanderWasTaken** [comment]
> Does it also happen after a restart?

Nope. If I just restart my phone, the accent is the same as I set it in the app. The issue is only present after a system update.
